app/scripts/services/sql.js

```diff
@@ -92,6 +92,8 @@ angular.module('sql', [])
         data.args = args;
       }
 
+      isConsole = typeof isConsole !== 'undefined' ? isConsole : false;
```

Comment:
I know it's a pain in the ass to change all method calls to use the new signature, but imo the typeof check is not best practice. it also has the flaw, that you could pass any type that is not boolean which would break the `not` check